index.js

```diff
@@ -9,6 +10,15 @@ module.exports = createXHR
 createXHR.XMLHttpRequest = window.XMLHttpRequest || noop
 createXHR.XDomainRequest = "withCredentials" in (new createXHR.XMLHttpRequest()) ? createXHR.XMLHttpRequest : window.XDomainRequest
 
+forEach(["get", "put", "post", "patch", "head", "delete"], function(method) {
```

Comment:
(awthanks) 😊

We could remove the `forEach` module here in favor of `Array.prototype.forEach` when we're free of ie8. Of course, the ie8 version would continue to include this dependency.